tests/resources/test_resource_utils.py

```diff
@@ -0,0 +1,54 @@
+from unittest.mock import patch
+
+from gnomad.resources import resource_utils
+
+
+class TestTableResource:
+    @patch("hail.read_table")
+    def test_read_table(self, read_table):
+        resource = resource_utils.TableResource("gs://my-bucket/table.ht")
```

Comment:
When testing, do we need a path to a valid file?

Reply:
Not really. In this test, [patch](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) replaces the real `hl.read_table` function with a mock that does nothing. However, if the relevant Hail function isn't patched in a test, I guess it would be preferable to attempt reading from a bucket we control. Will change to a gnomAD bucket.